src/System.Private.CoreLib/shared/System/Threading/ManualResetEventSlim.cs

```diff
@@ -449,7 +428,7 @@ public void Wait(CancellationToken cancellationToken)
         /// false.</returns>
         /// <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="timeout"/> is a negative
         /// number other than -1 milliseconds, which represents an infinite time-out -or- timeout is greater
-        /// than <see cref="System.Int32.MaxValue"/>.</exception>
+        /// than <see cref="System.int.MaxValue"/>.</exception>
```

Comment:
`System.int` is not a valid thing. This should stay as System.Int32.

Reply:
Could you please fix this in all CoreLib files in CoreCLR while you are on it?

```
  C:\coreclr\src\System.Private.CoreLib\shared\System\Threading\SpinWait.cs(258):        /// <see cref="System.int.MaxValue"/>.</exception>
  C:\coreclr\src\System.Private.CoreLib\src\System\Threading\ManualResetEventSlim.cs(441):        /// than <see cref="System.int.MaxValue"/>.</exception>
  C:\coreclr\src\System.Private.CoreLib\src\System\Threading\ManualResetEventSlim.cs(470):        /// than <see cref="System.int.MaxValue"/>.</exception>
  C:\coreclr\src\System.Private.CoreLib\src\System\Threading\SpinLock.cs(222):        /// than <see cref="System.int.MaxValue"/> milliseconds.
  C:\coreclr\src\System.Private.CoreLib\src\System\Threading\Tasks\Task.cs(2721):        /// <see cref="System.int.MaxValue"/>.
  C:\coreclr\src\System.Private.CoreLib\src\System\Threading\Tasks\Task.cs(4524):        /// <see cref="System.int.MaxValue"/>.
  C:\coreclr\src\System.Private.CoreLib\src\System\Threading\Tasks\Task.cs(4911):        /// <see cref="System.int.MaxValue"/>.
```

Reply:
yeah sure i will do it